modular_skyrat/modules/gladiator/code/modules/mob/living/simple_animal/hostile/megafauna/markedone.dm

```diff
@@ -376,11 +372,33 @@
 	boned.throw_at(target, 7, 3, thrower = src)
 	QDEL_IN(boned, 3 SECONDS)
 
-/mob/living/simple_animal/hostile/megafauna/gladiator/proc/swordslam() //this kills the crab
-	wendigo_slam(src, 5, 1 SECONDS, 8)
+/mob/living/simple_animal/hostile/megafauna/gladiator/proc/ground_pound(range, delay, throw_range) //copied and pasted wendigo slam attack, used for more slowly radiating AOE slams 
+	var/turf/origin = get_turf(src)
+	if(!origin)
+		return
+	var/list/all_turfs = RANGE_TURFS(range, origin)
+	for(var/sound_range = 0 to range)
+		playsound(origin,'sound/effects/bamf.ogg', 600, TRUE, 10)
+		for(var/turf/stomp_turf in all_turfs)
+			if(get_dist(origin, stomp_turf) > sound_range)
+				continue
+			new /obj/effect/temp_visual/small_smoke/halfsecond(stomp_turf)
+			for(var/mob/living/target in stomp_turf)
+				if(target == src || target.throwing)
+					continue
+				to_chat(target, span_userdanger("[src]'s ground slam shockwave sends you flying!"))
+				var/turf/thrownat = get_ranged_target_turf_direct(src, target, throw_range, rand(-10, 10))
+				target.throw_at(thrownat, 8, 2, null, TRUE, force = MOVE_FORCE_OVERPOWERING, gentle = TRUE)
+				target.apply_damage(20, BRUTE, wound_bonus=CANT_WOUND)
+				shake_camera(target, 2, 1)
+			all_turfs -= stomp_turf
+		sleep(delay)
+
+/mob/living/simple_animal/hostile/megafauna/gladiator/proc/swordslam() //large radius but slow-to-move radiating ground slam
+	ground_pound(5, 1 SECONDS, 8)
 
-/mob/living/simple_animal/hostile/megafauna/gladiator/proc/stomp()
-	wendigo_slam(src, 2, 0.5 SECONDS, 3)
+/mob/living/simple_animal/hostile/megafauna/gladiator/proc/stomp() //short range slam with faster shockwave travel
```

Comment:
documentation please (`/// documentation here` on the line before your proc (still keep at least one empty newline between each proc))